easybuild/easyconfigs/f/FreeFem++/FreeFem++-3.58_ebvars.patch

```diff
@@ -0,0 +1,72 @@
+# Pick up EB provided ScaLAPACK and FFTW in PETSc install 
+# August 23rd 2018 by B. Hajgato (Free University Brussels - VUB)
+--- freefem++-3.58/download/ff-petsc/Makefile.orig	2018-01-03 07:55:10.000000000 +0100
++++ freefem++-3.58/download/ff-petsc/Makefile	2018-08-25 13:21:53.409212134 +0200
+@@ -51,7 +51,7 @@
+ ifeq ($(MPICC),$(W_MPICC))
+   WITH_MPI_DIR=
+ else
+-  WITH_MPI_DIR="--with-mpi-dir='$(MPI_DIR)'"
++  WITH_MPI_DIR=--with-mpi-dir='$(MPI_DIR)'
```

Comment:
@hajgato Why this change?

Reply:
Beacuse without it the build failed. I did not look into the details. But does it really matters?

Reply:
It's weird that it's needed, I was just wondering why that change was made. I guess it doesn't matter too much as long as it fixes a problem. ;)

Reply:
I think it is the nature of `PETSc` build, it is a `Python` build system. If I run the `PETSc` installation script by hand (outside `EasyBuild`), it works with the double-quotation marks, using `EasyBuild` I had to remove it. (`PETSc` script was complaining something like cannot configure with `--with-mpi-dir` value.) So probably because it is a `Python` in a `Python`